### PR TITLE
Feature/profile tie to be

### PR DESCRIPTION
**Pull request by pernstrong**
#### What's this PR do?
Adds edit profile component and ties to BE database.
#### Where should the reviewer start?
EditProfile.js
#### How should this be manually tested?
Sign up, login and try to update your info. Will need to check the psql database to confirm changes.
#### Any background context you want to provide?
n/a
#### What are the relevant tickets?
n/a
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a

### CODE REVIEW

File: code/api/src/modules/user/mutations.js [hunk 1]
```diff
@@ -2,8 +2,8 @@
 import { GraphQLString, GraphQLInt } from 'graphql'
 
 // App Imports
-import { UserType } from './types'
-import { create, remove } from './resolvers'
+import { UserType, UserType1 } from './types'
```
Comment (DavidTTran) [inline code/api/src/modules/user/mutations.js:5]: This variable name is going to have to change in the future. I recommend `UserUpdateType`

File: code/api/src/modules/user/resolvers.js [hunk 2]
```diff
@@ -77,3 +77,16 @@ export async function remove(parentValue, { id }) {
 export async function getGenders() {
   return Object.values(params.user.gender)
 }
+
```
Comment (raymondknguyen) [inline code/api/src/modules/user/resolvers.js:80]: Great job! I wonder if we need an 'auth' for future purposes

File: code/web/src/modules/user/EditProfile.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    const [ description, updateDescription ] = useState()
+    // const [ image, updateImage ] = useState()
+
+    // handle picture upload
+    // const handleSubmit = e => {
+    //     e.preventDefault()
+    //     const pic = fileInput.current.files[0]
+    //     // console.log(image)
+    //     updateImage(pic)
+    // }
```
Comment (raymondknguyen) [inline code/web/src/modules/user/EditProfile.js:30]: can we get rid of this?

**Review by raymondknguyen — APPROVED** [review]

File: code/web/src/modules/user/EditProfile.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+        clearInputs()
+    }
+
+    const clearInputs = () => {
+        updateName('')
+        updateEmail('')
+        updateShippingAddress('')
+        updateDescription('')
+    }
```
Comment (DavidTTran) [inline code/web/src/modules/user/EditProfile.js:54]: This clearInputs is a little redundant isn't it? If the user is successfully updated - we'd want to redirect them back to the user/profile path

File: code/api/src/modules/user/mutations.js (hunk 1 above)
Comment (nickstaylor) [reply code/api/src/modules/user/mutations.js:5]: Great job with digging into the code examples to get the correct User Type.

File: code/web/src/modules/user/EditProfile.js (hunk 4 above)
Comment (nickstaylor) [reply code/web/src/modules/user/EditProfile.js:54]: It's always a good idea to clear inputs on submit

**Review by nickstaylor — APPROVED** [review]

**Pull request merged**